Add button to terminate BSP Server

tpasternak opened the pull request:
The button calls command defined in .bsp/bloop.json/argvExit
field. Typically it's just bloop exit. If the field is empty,
a warning is logged.

Commit:
Add button to terminate BSP Server

The button calls command defined in `.bsp/bloop.json/argvExit`
field. Typically it's just `bloop exit`. If the field is empty,
a warning is logged.

jastice commented:
sorry, this is going to need another rebase after my recent changes are actually merged -_-

tpasternak commented:
ok, np

jastice commented:
Do you think it's possible to only add the terminate button to the widget when there is an `argvExit`?

tpasternak commented:
Yes, I'll do it

Commit:
Hide "Terminate server" button if not supported

Server termination is supported only if the field `argvExit` is
defined in ".bsp/*.json" file.

jastice commented:
manually merged with minor updates via 
ebf0e9ce026f98b665cada9191f7e6ed0e85356c 
8cc65e96e23c62599a878251ee9cca016a134df1 
06eca3fffb3573e42d2104e3e5fbf9119acdcb68
